Week2/week2.txt

```diff
@@ -0,0 +1,35 @@
+1. create table stud*/
+ create table stud
+ (
+ id int primary key,
+ name text
+ );
+
+2. create table student_detail*/
+
+create table stud_detail
+(
+sid int,
+city text,
+mobile_no int,
+foreign key (sid) references stud(id)
+);
+
+---- insert queries of student -------
+
+ insert into stud values(1,'jay'),(2,'happy'),(3,'om'),(4,'sai'),(5,'karan'),(6,'vatsal'),(7,'meet'),(8,'harsh'),(9,'amit'),(10,'hiren');
+
+-------- insert record of stud_detail--------
+
+ insert into stud_detail values(1,'surat',9879435713),(2,'surat',8320839502),(3,'bardoli',1234567890),(4,'bardoli',9874561230),(5,'vyara',6320563212),(6,'baruch',7896452315),(7,'vyara',9909888561),(8,'surat',9320934560),(9,'bardoli',7946235698),(10,'surat',9656142389);
+
+/* queries*/
+1. display student whose come from surat.
+---> select s.id ,s.name from stud s join stud_detail sd on s.id=sd.sid where city='surat';
+2. display student detail whose city name start with 'b'.
+---> select s.id ,s.name from stud s join stud_detail sd on s.id=sd.sid where city like 'b%';

```

Comment:
I didn't use "join" in the code snippet you provided earlier. The code simply reads data from a CSV file and assigns values to variables. If you're referring to the term "join," it might be used in a different context in SQL queries or string operations to combine data from multiple sources.